Sources/FunctionalTableData/TableView/FunctionalTableData+UITableViewDelegate.swift

```diff
@@ -255,5 +255,28 @@ extension FunctionalTableData {
 				cellConfig.actions.contextMenuConfiguration?.previewContentCommitter?(animator.previewViewController)
 			}
 		}
+		
+		/// Requires iOS 13.0+
+		/// Whether multi selection is available when editing mode is enabled.
+		/// Based on the tableView's `allowsMultipleSelectionDuringEditing` property and `CellActions`'s `shouldBeginMultiSelectAction` result.
+		/// https://developer.apple.com/documentation/uikit/uitableviewdelegate/selecting_multiple_items_with_a_two-finger_pan_gesture
+		@available(iOS 13.0, *)
+		public func tableView(_ tableView: UITableView, shouldBeginMultipleSelectionInteractionAt indexPath: IndexPath) -> Bool {
+			let cellConfig = data.sections[indexPath]
+			
+			let shouldBeginMultiSelect = cellConfig?.actions.shouldBeginMultiSelectAction?() ?? false
+			return tableView.allowsMultipleSelectionDuringEditing && shouldBeginMultiSelect
+		}
+		
+		/// Requires iOS 13.0+
+		/// Called when a multi item selection gesture has begun.
+		/// - Handler forwarded to `CellActions` via `beginMultiSelectAction`
+		@available(iOS 13.0, *)
+		public func tableView(_ tableView: UITableView, didBeginMultipleSelectionInteractionAt indexPath: IndexPath) {
+			tableView.setEditing(true, animated: true)
```

Comment:
Is it required that the tableView enter edit mode whenever using multi select?

Reply:
Yes! 
Also, in order for it to work, the function `public func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool` needs to return true, otherwise the tableView will not have the checkbox accessory view to the left of the cell. Which I'm planning on adding either our next PR :)